content/integrations/statsd.md

```diff
@@ -12,7 +12,9 @@ Enable the Datadog StatsD integration to:
   * Monitor uptime and availability of a StatsD server
   * Collecting metrics from a StatsD server's admin interface
 
-**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. This integration is intended for monitoring external StatsD servers, and is not needed to send metrics to Datadog using the StatsD protocol.**
+**Please note the Datadog Agent includes [dogstatsd](http://docs.datadoghq.com/guides/dogstatsd/) which serves as a StatsD forwarder. If you want to send metrics from a statsd server, just point it to the forwarder listening on port 17123.**
```

Comment:
Well, I've just read this like three times and I'm still not sure what the current content or proposed edit are saying, or if what Ilan is saying is correct, so, take that as you will. :)

What is this about forwarding? Can't we just tell readers that this check monitors StatsD itself, and link them to the DogStatsD page if they want to instrument metrics?

"Monitor the availability of your StatsD servers and track how many metrics they're processing.

This Agent check does not help you instrument your application metrics; it only monitors the status of StatsD itself. See [our DogStatsD guide] to learn how to instrument application metrics."

^ Though this doesn't cover the forwarding thing. I'm not sure myself how that's supposed to work - is it documented somewhere?